PDF/CMakeLists.txt

```diff
@@ -94,23 +98,20 @@ POCO_SOURCES(SRCS hpdf
 # with an unbundled version of libpng
 POCO_SOURCES(SRCS libpng
 	src/png.c
-	src/pngerror.c
-	src/pnggccrd.c
-	src/pngget.c
-	src/pngmem.c
-	src/pngpread.c
-	src/pngread.c
-	src/pngrio.c
-	src/pngrtran.c
-	src/pngrutil.c
-	src/pngset.c
-	src/pngtest.c
-	src/pngtrans.c
-	src/pngvcrd.c
-	src/pngwio.c
-	src/pngwrite.c
-	src/pngwtran.c
-	src/pngwutil.c
+    src/pngerror.c
+    src/pngget.c
+    src/pngmem.c
+    src/pngpread.c
+    src/pngread.c
+    src/pngrio.c
+    src/pngrtran.c
+    src/pngrutil.c
+    src/pngset.c
+    src/pngtrans.c
+    src/pngwio.c
+    src/pngwrite.c
+    src/pngwtran.c
+    src/pngwutil.c
```

Comment:
Unnecessary change of white space (tabs to spaces).

Is the list of source file shorter by three files intentionally or by accident?